Resources/Prototypes/Roles/Jobs/Security/security_cadet.yml

```diff
@@ -6,8 +6,10 @@
   requirements:
     - !type:DepartmentTimeRequirement
       department: Security
-      time: 7200
+      time: 14400
       inverted: true # stop playing intern if you're good at security!
+    - !type:OverallPlaytimeRequirement
+      time: 14400
```

Comment:
Why does a cadet need 4 hours of playtime?

Reply:
Probably to prevent entirely new players from playing sec.